src/neo/Network/P2P/Payloads/GetHeadersPayload.cs

```diff
@@ -0,0 +1,37 @@
+using Neo.IO;
+using System;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class GetHeadersPayload : ISerializable
```

Comment:
I think this is the same as `GetBlockDataPayload`. Maybe we can use only one payload. It can be renamed to `GetBlockByIndexPayload`.

Reply:
Yes. Renaming it will make it easier to understand.

Reply:
Done